docs/platforms/javascript/guides/react/features/component-names.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: React Component Names
+excerpt: ""
+description: "Learn how Sentry's React SDK allows you to monitor your components."
+---
+
+Sentry helps you capture your React components and unlock additional insights in your application. You can set it up to use React component names instead of selectors.
```

Comment:
```suggestion
Sentry helps you capture your React components and unlock additional insights in your application. You can set up your Sentry configuration to use React component names instead of selectors, whenever they are available.
```